pages/people/[id]/record.jsx

```diff
@@ -28,7 +28,7 @@ const CasesPage = ({ query }) => {
         placeHolder="Choose one"
         onChange={(value) => setUrl(value)}
       />
-      <LinkButton label="Load form" route={`${url}?id=${query.id}`} />
+      <LinkButton label="Load form" route={url} id={query.id} />
```

Comment:
instead of `id` can we have it a bit more generic? like `internalQuery={`id=${query.id}`}`?
The `LinkButton` should stay as generic as possible.